contrib/plugins/plugin-clair/clair.go

```diff
@@ -69,12 +69,41 @@ func (d ClairPlugin) Run(a plugin.IJob) plugin.Result {
 	_ = plugin.SendLog(a, "Running analysis")
 	analysis := clair.Analyze(image, manifest)
 
-	json, err := jsoniter.Marshal(analysis)
-	if err != nil {
-		_ = plugin.SendLog(a, "Unable to push image on Clair: %s", err)
+	_ = plugin.SendLog(a, "Creating report")
+
+	var vulnerabilities []sdk.Vulnerability
+	summary := make(map[string]int64, 0)
+	if analysis.MostRecentLayer().Layer != nil {
+		for _, feat := range analysis.MostRecentLayer().Layer.Features {
+			for _, vuln := range feat.Vulnerabilities {
+				v := sdk.Vulnerability{
+					Version:     feat.Version,
+					Component:   feat.Name,
+					Description: vuln.Description,
+					Link:        vuln.Link,
+					FixIn:       vuln.FixedBy,
+					Severity:    sdk.ToVulnerabilitySeverity(vuln.Severity),
+					Origin:      feat.AddedBy,
+					CVE:         vuln.Name,
+					Title:       fmt.Sprintf("%s %s", feat.Name, feat.Version),
+				}
+				vulnerabilities = append(vulnerabilities, v)
+
+				count, _ := summary[v.Severity]
```

Comment:
should write count := summary[v.Severity] instead of count, _ := summary[v.Severity]

Reply:
fixed